### ISSUE TITLE
ITV hub says "invalid page" when using 'use current webpage'

### ISSUE THREAD

**Issue by Iain-F** [Ascoware/get-iplayer-automator#121]
Thanks for GIA.

Updated to Version 1.10.1 (20180313001) today and now get an "invalid page" dialog when trying to 'use current webpage' to download an ITV program. The ITV page is the front most in the browser. So not sure what has changed
Cheers

Iain

![image](https://user-images.githubusercontent.com/37370754/37402585-157b8a68-2784-11e8-8592-f1616591025c.png)

**Comment by Lordsnooty05** [comment]
Iain please see other threads about this issue about this, Scott knows about it and he is looking into it. Try to read the other threads before starting a new issue....this is being dealt with

**Comment by skovatch** [comment]
I have a quick fix for this one, and will push an update today. Searchable listings will take some more time, but I have made progress.

**Comment by Iain-F** [comment]
Hi,
apologies. Not trying to cause more work. Did scan recent post titles before starting new thread. In addition the update mentioned an ITV issue in the dialog box noting the availability of an update, so I thought that this might be new...
Cheers
Iain

**Comment by Iain-F** [comment]
Hi Scott
thanks for the rapid update. 
Unfortunately I'm still being unsuccessful with a log window message as attached. I can't see where the "other formats" section is. I have BBC TV, BBC radio and ITV formats sections but no "other formats". What am I missing?
Cheers
Iain
![image](https://user-images.githubusercontent.com/37370754/37433888-98cd6da4-27d5-11e8-84ea-a5478765c41e.png)

**Comment by skovatch** [comment]
Now you are running into the other big ITV issue in that not every show has a streamable version. I’m trying to find time to work on that as it’s a major change.

**Comment by Iain-F** [comment]
Oh, OK.
I'm just surprised as all other episodes of Endeavour have bee downloadable without problems. I guess ITV are continually trying to make things more difficult, (in that there must be a 'streamable' version for playback on my computer; I've checked that the program in question does stream)
Thanks again for all your work on GIA.
Cheers
Iain
